lib/lndclient/LndClient.ts

```diff
@@ -177,6 +180,14 @@ class LndClient extends SwapClient {
     return this.maximumOutboundAmount;
   }
 
+  public maximumChannelOutboundCapacity = () => {
```

Comment:
This can be a true getter instead of a method with `public get maxOutboundAmount() {...` (assuming we use my suggested naming). I think it's a bit neater/more consistent that way.